### ISSUE TITLE
`-z` the only duplicates option for `mimeo` is not correctly implemented.

### ISSUE THREAD

**Issue by kopardev** [CCBR/spacesavers2#71]
Observations:
- duplicates are 2 and only 2 files are shown .. expectation is 3 files as original file should always be reported. This is likely when the original is owned by a different uid.
- `-z` should only filter files out which do not have duplicates .. not filter out the original file as well

Scenario:
- original file "a"
- copied to "b"
- "b" is then hardlinked to "c"
- "c" is then copied to "d"
- "d" is hardlinked to "e" and "f"

What will happen:
- "a" through "f" all will have the same top_hash and bottom_hash, same size and account for 3 inodes
- Hence, this should be 1 original + 2 duplicates even though you have 6 total files.
- basically duplicate 2 duplicate inodes account for 6 files... so dup_bytes should by X 2 while `usurp` should replace 5 files with hard links
